### ISSUE TITLE
ldconfig files deleted upon RPM upgrade/reinstall

### ISSUE THREAD

**Issue by xw285cornell** [RadeonOpenCompute/ROCm#534]
We just found an issue when upgrading or reinstalling the RPMs. The post-install script of the rpm says this: 

```
postuninstall scriptlet (using /bin/sh):
rm -f /etc/ld.so.conf.d/hsa-rocr-dev.conf && ldconfig
```

But when one upgrades (or re-installs) a RPM, that first installs the new one, runs the post-install scriplet, which will create the file, then it removes the old version, and runs the postuninstall scriplet. Therefore the ldconfig file is deleted. 

We think the postrun scriptlet should be something like: 

``` 
if [ $1 -eq 0]; then
  # the old script, like rm -f /etc/ld....
fi
```

**Comment by kentrussell** [comment]
Yeah this is something we hit with the kernel as well, and was incredibly annoying to try to figure out. We used the same resolution that you suggested though. I'll see if we can get someone on the ROCr team to take a look at it.

**Comment by kentrussell** [comment]
I've got a fix internally. Won't hit 1.9.1 but should be in 1.9.2 .

**Comment by kentrussell** [comment]
Issue addressed since 1.9.2
